kiwmi/luak/kiwmi_keyboard.c

```diff
@@ -16,6 +16,29 @@
 #include "input/keyboard.h"
 #include "luak/kiwmi_lua_callback.h"
 
+static int
+l_kiwmi_keyboard_configure(lua_State *L)
+{
+    struct kiwmi_keyboard *keyboard =
+        *(struct kiwmi_keyboard **)luaL_checkudata(L, 1, "kiwmi_keyboard");
+
+    struct xkb_rule_names rules = {
+        .rules   = luaL_checkstring(L, 2),
+        .model   = luaL_checkstring(L, 3),
+        .layout  = luaL_checkstring(L, 4),
+        .variant = luaL_checkstring(L, 5),
+        .options = luaL_checkstring(L, 6),
+    };
+    wlr_log(WLR_INFO, " %s ", lua_tostring(L, 2));
```

Comment:
Maybe something like "changing keymap ..." instead, also logging the other values.

Reply:
I removed the logging, it was some leftover. Or do you think we need some logging here.